### PR TITLE
refractor: improved the layout of saved experiment activity

### LINKED ISSUE #849: Improve the layout of the saved experiment activity
**Actual Behaviour**

The layout of the saved experiment activity is too basic

**Expected Behaviour**

The layout should have icons which makes it look more interactive

**Screenshot of the issue**
image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes #849

Changes: Improved the layout of the saved experiment by adding interactive icons

Screenshots for the change: 
image

APK for testing:
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1996166/app-debug.zip)

### CODE REVIEW

**Comment by Avjeet** [comment]
@CloudyPadmal @akarshan96 plz review

File: app/src/main/res/layout/saved_experiments_header_holder.xml [hunk 1]
```diff
@@ -2,31 +2,52 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:minHeight="48dp">
+    android:minHeight="48dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/saved_experiments_header_holder.xml]: You've missed it here

File: app/src/main/res/layout/saved_experiments_header_holder.xml [hunk 2]
```diff
@@ -2,31 +2,52 @@
...
-        android:src="@drawable/ic_arrow_drop_down_black_24dp"
-        android:contentDescription="@string/drop_down_arrow" />
+        <TextView
+            android:id="@+id/experiment_header_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginLeft="@dimen/margin_10dp"
+            android:layout_marginStart="@dimen/margin_10dp"
+            android:textSize="18sp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/saved_experiments_header_holder.xml]: and here

**Comment by Avjeet** [comment]
changes done @CloudyPadmal

**Comment by CloudyPadmal** [comment]
@Avjeet can you check why the Travis build is failing and fix that?

**Comment by Avjeet** [comment]
@cloudy padmal travis build resolved

**Comment by abhinavraj23** [comment]
@harsh-2711 @Vikum94 Please review this PR so that it can be merged.

**Comment by mariobehling** [comment]
No longer Saved Exp. in the app.

**Pull request closed without merging**